docs/mpcnc/belts.md

```diff
@@ -4,17 +4,18 @@
 
 - Move the gantry all the way to the lower left corner.
 - Paying attention to which way the cable tie heads are facing (for that slick clean look).
-- Tie one end of the belt just enough to fit a cable tie through the loop, teeth facing each other and meshed, teeth for the rest of the belt facing down. Pull tight.
-- Loop the next tie through the belt loop, and the corner piece. Leave this one as loose as possible (1″-1.5″ (25-35mm) gap).
-
+- For each belt, the first cable tie will be setup very loose, so that it can be tightened after the belt is routed, and the cable tie on the opposite side is tightened.
```

Comment:
I don't think "very loose" or "as loose as possible" is right. Firstly, because the longer the zip ties, the more spring they will have. Secondly, because you can't easily make the belts longer.